contrib_bots/lib/ConverterBot/docs.md

```diff
@@ -0,0 +1,69 @@
+# Converter bot
+
+This bot allows users to perform conversions for various measurement units.
+
+## Usage
+
+Run this bot as described in [here](http://zulip.readthedocs.io/en/latest/bots-guide.html#how-to-deploy-a-bot).
+
+Use this bot with the following command
+
+`@convert <number> <unit_from> <unit_to>`
+
+This will convert `number`, given in the unit `unit_from`, to the unit `unit_to`
+and print the result.
+
+* `number` can be any floating-point number, e.g. 12, 13.05, 0.002.
+* `unit_from` and `unit_to` are two units from [the following](#supported-units) table.
+* `unit_from` and `unit_to` can be preceded by [these](#supported-prefixes) prefixes.
+
+### Supported units
+
+| Category | Units |
+| ----------------- | ----- |
+| Area | square-centimeter (cm^2, cm2), square-decimeter (dm^2, dm2), square-meter (m^2, m2), are (a), hectare (ha), square-kilometer (km^2, km2), square-inch (in^2, in2), square-foot (ft^2, ft2), square-yard (y^2, y2), acre (ac), square-mile (mi^2, mi2) |
```

Comment:
hm, I understand your point. It would not be by size though, but sure, can put it in this order.